specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -575,23 +575,25 @@
 					"$ref": "./examples/SearchServiceGetSkillset.json"
 				}
 			},
-			"description": "Retrieves a skillset from Azure Search.",
+			"description": "Retrieves a Cognitive Search skillset in an Azure Search service.",
 			"externalDocs": {
-				"url": "https://docs.microsoft.com/en-us/azure/search/cognitive-search-tutorial-blob"
+				"url": "https://docs.microsoft.com/azure/search/cognitive-search-tutorial-blob"
 			},
-			"parameters": [{
-				"name": "skillsetsName",
-				"in": "path",
-				"required": true,
-				"type": "string",
-				"description": "The name of the skillset to retrieve."
-			},
-			{
-				"$ref": "#/parameters/ClientRequestIdParameter"
-			},
-			{
-				"$ref": "#/parameters/ApiVersionParameter"
-			}],
+			"parameters": [
+				{
+					"name": "skillsetName",
+					"in": "path",
+					"required": true,
+					"type": "string",
+					"description": "The name of the skillset to retrieve."
+				},
+				{
+					"$ref": "#/parameters/ClientRequestIdParameter"
+				},
+				{
+					"$ref": "#/parameters/ApiVersionParameter"
+				}
+			],
 			"x-ms-request-id": "request-id",
 			"responses": {
 				"200": {
```

Comment:
Each response should also have a description. See [the Azure Search management plane spec](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/search/resource-manager/Microsoft.Search/stable/2015-08-19/search.json) for examples.

Reply:
Added